### DISCUSSION TITLE
Gestion des équipes

### DISCUSSION THREAD

**Discussion by opetitprestataire** [IGNF-Ma-carte/.github#7] [Général]
Bonjour,
Y a-t-il un tutoriel sur les équipes car nous ne parvenons pas à voir les cartes des membres de l'équipe dans l'espace ?
Merci beaucoup

**Comment by Viglino** [comment]
Bonjour,

La documentation sur les équipes est en cours de rédaction.
Un premier jet est disponible sur le chatbot (version béta) : https://ignf-ma-carte.github.io/chatbot/?q=%C3%A9quipes

Seuls les membres de l'équipe peuvent voir les cartes de l'équipe (hors cartes dans l'atlas) et cela va dépendre de leur statut (privé) et du rôle de l'utilisateur dans l'équipe (membre, éditeur, propriétaire). 
[voir ici](https://ignf-ma-carte.github.io/chatbot/?q=%23equipe%2FQui%20peut%20voir%20les%20cartes%20d%27une%20%C3%A9quipe).

Si vous constatez des bugs ou si vous avez encore des incertitudes sur le fonctionnement, n'hésitez pas à nous les remonter pour améliorer la documentation.

@+

**Reply by opetitprestataire** [reply]
Merci beaucoup !
